packages/config/config/devices/0x0258/motion_sensor_2.json

```
@@ -0,0 +1,158 @@
+{
+	"manufacturer": "Shenzhen Neo Electronics Co., Ltd.",
+	"manufacturerId": "0x0258",
+	"label": "Motion Sensor 2",
+	"description": "PIR Motion Sensor 2",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x008d"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"1": {
+			"label": "Sensitivity Level Setting",
```

```suggestion
			"label": "Sensitivity Level",
```
Also, the description is repetitive and should explain the values (e.g. 8 = low, 255 = high)